### PR TITLE
Added favicons and default title text

### PR DESCRIPTION
**Pull request by joe-thom**

### CODE REVIEW

File: packages/electron/app/package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
   },
   "resolutions": {
     "electron": "3.1.0",
-    "debug": "^2.6.9"
+    "debug": "^2.6.9",
```
Comment (paulvarache) [inline packages/electron/app/package.json]: We'll have to revert that change. It causes issues with macos builds

File: packages/web/package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "livereload": "^0.7.0",
     "rimraf": "^2.6.2",
     "serve-static": "^1.13.2",
+    "sharp": "^0.23.0",
```
Comment (paulvarache) [inline packages/web/package.json]: We need to make sure that's the same version as the other packages. Sharp can't have multiple versions as it uses a dynamically linked library

File: packages/electron/app/package.json (hunk 1 above)
Comment (joe-thom) [reply packages/electron/app/package.json]: Are you referring to `debug` and `xpc-connection` or just `debug`?

File: packages/web/package.json (hunk 2 above)
Comment (joe-thom) [reply packages/web/package.json]: Made sure all versions of sharp are 0.22.1 👍

**Comment by joe-thom** [comment]
Added some extra changes also, mainly regarding the web/build file - one being a check for the config value and the second check is to not render out the favicon template if one isn't provided

**Pull request merged**